README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running an Elasticsearch cluster
+
+## Services deployed:
+
+- `es-master`: Elasticsearch nodes, part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.
+- Justwatch [Exporter]: Exposes Prometheus metrics on the cluster status.
+- es-client: Rancher cattle DNS service, which points the `es-client` domain name to `es-master`, for backwards compatibility with existing services.
+
+## Deployment
+
+Deployment is driven by Travis (Dev and PR review deployments) and TeamCity (staging and production deployments). We use the NHS.UK [CI-Deployment] repo for deployment to the Rancher environment, this sets the following ENV variables.
+
+## Environment variables
+
+| Variable | Description | Default | 
+| -------- | ----------- | ------- |
+| `ES_MIN_MASTER_NODES` | The minimum nodes required to run the cluster | `2` |
+| `ES_DATA_VOL` | The name of the local volume on the host to use | `esdata` |
+
+## How it works
+
+The number of containers running is determined by the number of hosts that have the `escluster` flag set in the Rancher environment. The minimum number should be `3`.
+
+When the containers start they do a DNS A record lookup for `es-master` to get the current hosts in the cluster, ElasticSearch uses this information to build the cluster and for communication.
+
+## How to use the cluster
+
+The cluster is currently setup to only be available inside of the Rancher environment, so your application will have to be deployed to that environment also.
```

Comment:
I had to do some digging to remind myself of the distinction between _stacks_ and _environments_ so might be useful to link to http://rancher.com/docs/rancher/latest/en/environments/

Reply:
I think that is outside the scope of this documentation.